core/src/main/scala/cats/data/WriterT.scala

```diff
@@ -21,6 +21,11 @@ final case class WriterT[F[_], L, V](run: F[(L, V)]) {
       functorF.map(run) { z => (z._1, fn(z._2)) }
     }
 
+  def contramap[Z](fn: Z => V)(implicit F: Contravariant[F]): WriterT[F, L, Z] =
+    WriterT {
+      F.contramap(run) { z => (z._1, fn(z._2)) }
+    }
```

Comment:
I wouldn't have thought this would be terribly useful (but I could be totally wrong), let me know what you think.